.pre-commit-config.yaml

```diff
@@ -138,14 +137,13 @@ repos:
         always_run: true
         args: [audit, --json, --ignore-code=CVE-2019-8341]
```

Comment:
Please move the `audit` string into the `entry` key

Reply:
Also remove the `types` key and replace with `files: ^(.*/)?(poetry\.lock|pyproject\.toml)$`